app/src/main/res/values/strings.xml

```diff
@@ -4,5 +4,16 @@
     <string name="make_note">Make a journal note</string>
     <string name="done">Done</string>
     <string name="read_more">Read more</string>
+    <string name="Date">Tuesday, April 18th</string>
+    <string name="happy_face_icon">happy face icon</string>
+    <string name="gratefulness">Today I\'m grateful for...</string>
+    <string name="today_best_part">Best part of today was...</string>
+    <string name="my_note">My note:</string>
+    <string name="add_photo">add photo</string>
+    <string name="add_location">add location</string>
+    <string name="done_btn">done_btn</string>
+    <string name="grateful_edit">Insert what are you grateful for today</string>
```

Comment:
maybe instead of 'insert' lets make it 'write' or 'type', insert is very cold and technical hahaha